File: cpp/3rd_Week/MariMencoba3/Semen.cpp
#include "BahanBangunan.cpp"

class Semen: public BahanBangunan {
    private:
        string namaSemen, kodePabrik, kualitas, harga;

//    constructor
    public:
        Semen(){}

//    set and get emthod
    string getNamaSemen() {
        return namaSemen;
    }

    void setNamaSemen(string namaSemen) {
        this->namaSemen = namaSemen;
    }

    string getKodePabrik() {
        return kodePabrik;
    }

    void setKodePabrik(string kodePabrik) {
        this->kodePabrik = kodePabrik;
    }

    string getKualitas() {
        return kualitas;
    }

    void setKualitas(string kualitas) {
        this->kualitas = kualitas;
    }

    string getHarga() {
        return harga;
    }

    void setHarga(string harga) {
        this->harga = harga;
    }

    ~Semen(){};
};

File: java/javadpbo/src/com/sem4dpbo/week3/marimencoba3/Semen.java
package com.sem4dpbo.week3.marimencoba3;

public class Semen extends BahanBangunan {
    private String namaSemen, kodePabrik, kualitas, harga;

//    constructor
    public Semen(){}

//    set and get emthod
    public String getNamaSemen() {
        return namaSemen;
    }

    public void setNamaSemen(String namaSemen) {
        this.namaSemen = namaSemen;
    }

    public String getKodePabrik() {
        return kodePabrik;
    }

    public void setKodePabrik(String kodePabrik) {
        this.kodePabrik = kodePabrik;
    }

    public String getKualitas() {
        return kualitas;
    }

    public void setKualitas(String kualitas) {
        this.kualitas = kualitas;
    }

    public String getHarga() {
        return harga;
    }

    public void setHarga(String harga) {
        this.harga = harga;
    }

}
